scientist-softserv/harvard-curiosity
# Rails617

**Upgrading to rails 6.1.7 to address a security concern**
* * *


# What does this Pull Request do?
Moves us to rails 6.1.7 to address an issue in 6.1.5

# How should this be tested?

Run locally and make sure everything comes up and works. I would also run in production mode (either locally or on dev) just to confirm no changes in that mode.

Can shell in container and run a db:migrate

# Test coverage
Yes/No: Are changes in this pull-request covered by:
- unit tests?
- integration tests?

# Interested parties
Tag (@ mention) interested parties
